electrumsv/wallet.py

```diff
@@ -2996,6 +2999,16 @@ async def _obtain_merkle_proofs_worker_async(self) -> None:
             await self._check_missing_proofs_event.wait()
             self._check_missing_proofs_event.clear()
 
+    async def fetch_raw_transaction_async(self, tx_hash: bytes, account: AbstractAccount) \
+            -> Optional[bytes]:
+        """Selects a suitable server and requests the raw transaction.
+
+        Raises `ServerConnectionError` if the remote server is not online (and other networking
+            problems).
+        Raises `GeneralAPIError` if a connection was established but the request errored.
+        """
+        return await request_transaction_data_async(self._network, account, tx_hash)
```

Comment:
Add a `TODO(1.4.0)` comment about how we intercept this call because the wallet will be funding it via petty cash and needs to wrap the call to apply checks and handling?